manifests/init.pp

```diff
@@ -61,6 +61,9 @@
   $session_lastvalidation                                        = 'session.lastvalidation',
   $proxy_server                                                  = undef,
   $proxy_type                                                    = undef,
+  String $mysql_connector_version                                = '5.1.47',
+  String $mysql_connector_install                                = '/opt/',
```

Comment:
This will always be an absolute path I assume, so we can use a more strict datatype:
```suggestion
  Stdlib::Absolutepath $mysql_connector_install                  = '/opt/',
```